From 9f42dfacb192757711b695a8c7ca456c1d84efac Mon Sep 17 00:00:00 2001
From: Shengjiu Wang <shengjiu.wang@freescale.com>
Date: Thu, 9 Apr 2015 19:47:19 +0800
Subject: [PATCH] MLK-10540: ARM: dts: imx7d-sdb: Enable 5k_PU for TXD in sai

Enable 5k_PU for TXD in sai, which can reduce the noise in mono playback.

Signed-off-by: Shengjiu Wang <shengjiu.wang@freescale.com>
---
 arch/arm/boot/dts/imx7d-sdb.dts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/imx7d-sdb.dts b/arch/arm/boot/dts/imx7d-sdb.dts
index 54fb68c..a7a9930 100644
--- a/arch/arm/boot/dts/imx7d-sdb.dts
+++ b/arch/arm/boot/dts/imx7d-sdb.dts
@@ -739,7 +739,7 @@
 				MX7D_PAD_SAI1_MCLK__SAI1_MCLK           0x1f
 				MX7D_PAD_ENET1_RX_CLK__SAI1_TX_BCLK     0x1f
 				MX7D_PAD_ENET1_CRS__SAI1_TX_SYNC	0x1f
-				MX7D_PAD_ENET1_COL__SAI1_TX_DATA0	0
+				MX7D_PAD_ENET1_COL__SAI1_TX_DATA0	0x30
 				MX7D_PAD_ENET1_TX_CLK__SAI1_RX_DATA0	0x1f
 			>;
 		};
@@ -748,7 +748,7 @@
 			fsl,pins = <
 				MX7D_PAD_SAI2_TX_BCLK__SAI2_TX_BCLK     0x1f
 				MX7D_PAD_SAI2_TX_SYNC__SAI2_TX_SYNC     0x1f
-				MX7D_PAD_SAI2_TX_DATA__SAI2_TX_DATA0    0
+				MX7D_PAD_SAI2_TX_DATA__SAI2_TX_DATA0    0x30
 				MX7D_PAD_SAI2_RX_DATA__SAI2_RX_DATA0    0x1f
 			>;
 		};
-- 
1.8.0

